config/routes.rb

```diff
@@ -1,15 +1,17 @@
 Myflix::Application.routes.draw do
   root to: 'pages#front'
 
-  get  '/sign_in',  to: 'sessions#new'
-  post '/sign_in',  to: 'sessions#create'
-  get  '/sign_out', to: 'sessions#destroy'
-  get  '/register', to: 'users#new'
-  get  '/home',     to: "videos#index", as: 'home'
-  get '/my_queue',  to: "queue_items#index"
-  post '/my_queue/update', to: "queue_items#update"
+  get  '/sign_in',           to: "sessions#new"
+  post '/sign_in',           to: "sessions#create"
+  get  '/sign_out',          to: "sessions#destroy"
```

Comment:
I would recommend using delete for #destroy, to be semantically right and also destructive routes should not be bookmarked.